src/components/BulkEnrollmentPage/CourseSearchResults.jsx

```diff
@@ -53,13 +89,15 @@ export const BaseCourseSearchResults = ({
         <DataTable
           columns={columns}
           data={searchResults?.hits || []}
-          itemCount={searchResults?.nbHits || 0}
```

Comment:
so we no longer default to 0, or is the defaulting done elsewhere?

Reply:
We're not even showing this component if there are no results right now. Which now that I think of it will not work as well once we've got filters. But I will handle that when I add filters. 😄

Reply:
(Putting a note on that ticket now.)